### PR TITLE
Updating Framework from AD review comments

### PR DESCRIPTION
**Pull request by dfedyk**

### CODE REVIEW

File: data-plane-framework/draft-ietf-detnet-data-plane-framework.xml [hunk 1]
```diff
@@ -635,6 +654,22 @@ CE1----EN1--------R1-------R2-------R3--------EN2-----CE2
...
+     of out-of-order packets that can be buffered and the delay difference of
+     arriving packets.  DetNet uses configuration of the number of maximum
+     number out-of-order packets and the maximum delay that out-of-order
+     packets can be held before being delivered. If the differential delay
+     between paths is large enough or the arrival rate of packets large enough
+     packets may be dropped instead of being reordered.  Likewise elimination
+     uses the sequence number to eliminate duplicate packets and large
+     differential delays combined with high numbers of packets may exceed the
+     ability of this elimination function to eliminate all duplicate packets.
+     Network engineering of DetNet services needs to take these factors into
```
Comment (louberger) [inline data-plane-framework/draft-ietf-detnet-data-plane-framework.xml]: s/Network engineering of/Path selection for

File: data-plane-framework/draft-ietf-detnet-data-plane-framework.xml [hunk 2]
```diff
@@ -1012,12 +1048,16 @@ Sub-Network        |  L2  |  | TSN  |  | UDP  |
         provide these functions along a flow's path through the
         network. See  <xref target="I-D.ietf-detnet-ip"/> and <xref
         target="control-management-requirements"/> for further
-        discussion of these requirements.
+        discussion of these requirements. Some forms of protection
+        may enforce packet loss or change
```
Comment (louberger) [inline data-plane-framework/draft-ietf-detnet-data-plane-framework.xml]: s/ensure/minimize

Comment (vb-balazs) [reply data-plane-framework/draft-ietf-detnet-data-plane-framework.xml]: Please clarify what we intend to say here.

**Review by louberger — CHANGES_REQUESTED** [review]
Please address comments made during han 27the meeting

File: data-plane-framework/draft-ietf-detnet-data-plane-framework.xml (hunk 2 above)
Comment (dfedyk) [reply data-plane-framework/draft-ietf-detnet-data-plane-framework.xml]: I believe you mean s/enforce/minimize/

File: data-plane-framework/draft-ietf-detnet-data-plane-framework.xml (hunk 1 above)
Comment (dfedyk) [reply data-plane-framework/draft-ietf-detnet-data-plane-framework.xml]: OK

**Review by louberger — APPROVED** [review]
changes look good

**Pull request merged**